cloud-run/src/vulnerability-scanning.js

```diff
@@ -0,0 +1,72 @@
+const exec = require('@actions/exec');
+const core = require('@actions/core');
+const notifySlack = require('../../slack-notify/src/slack-notify');
+
+// 1. download trivvy and install bin
+const installTrivy = async () => {
```

Comment:
Except for this new functionality, I believe the cloud-run action can run on Windows. If it's not possible to run it on Windows, or just too much work, maybe we can disable the Trivy features for Windows?